src/get-started/java-binding.md

```diff
@@ -180,11 +197,44 @@ current storage state when the transaction is executed. Exonum passes `Fork`
 as an argument – a view that allows performing modifying operations. A service
 schema object can be used to access data collections of this service.
 
+Also, `Transaction#execute` method may throw `TransactionExecutionException`
+which contains a transaction error report. This feature allows users to notify
+Exonum about an error in a transaction execution whenever one occurs.
+It may check the preconditions before executing a transaction and either
```

Comment:
It checks the preconditions...

Reply:
I'd keep it as is (or clarify), as a _transaction_ may (or may not) check the preconditions.